changelog.d/distr_opts_improvements_graceful_load_fallback.change.rst

```diff
@@ -0,0 +1,2 @@
+* When an entry point now fails to be loaded, it doesn't disrupt building process.
```

Comment:
This changelog entry needs work. First, the filename shouldn't have any text in it, only an issue number and scope (i.e. "change"). Second, the text references "an entry point", but doesn't talk about the context. This change only affects the loading of `setuptools.finalize_distribution_options`.